pom.xml

```diff
@@ -271,7 +271,8 @@
         <ironmq-version>3.0.5</ironmq-version>
         <ivy-version>2.5.2</ivy-version>
         <jackson-jq-version>1.0.0-preview.20240207</jackson-jq-version>
-        <jackson2-version>2.16.2</jackson2-version>
+        <jackson2-version>2.17.1</jackson2-version>
+        <jackson2-version-range>[2.17,2.18)</jackson2-version-range>
```

Comment:
This section is actually copied/pasted from the camel parent pom as a temporary solution, so could you please move this maven property after the tag `END: Maven Properties defining the version of 3rd party libraries used in Camel` at line 517 and I would rather expect a name like `camel.osgi.jackson2.version` to remain consistent with the rest